.gitlab-ci.yml

```diff
@@ -2,30 +2,97 @@
 include:
   - project: SocialGouv/gitlab-ci-yml
     file: /base_register_stage.yml
-    ref: v4.3.0
+    ref: v4.5.4
+  - project: SocialGouv/gitlab-ci-yml
+    file: /base_create_namespace_stage.yml
+    ref: v4.5.4
   - project: SocialGouv/gitlab-ci-yml
     file: /base_docker_helm_image_stage.yml
-    ref: v4.3.0
+    ref: v4.5.4
   - project: SocialGouv/gitlab-ci-yml
     file: /base_docker_kubectl_image_stage.yml
-    ref: v4.3.0
+    ref: v4.5.4
+  - project: SocialGouv/gitlab-ci-yml
+    file: /base_deploy_hpa_chart_stage.yml
+    ref: v4.5.4
+  #
+  - https://raw.githubusercontent.com/SocialGouv/emjpm/master/.gitlab-ci/extends.yml
   #
   - /.gitlab-ci/variables.yml
-  - /.gitlab-ci/stages/quality.yml
-  - /.gitlab-ci/stages/register.yml
-  - /.gitlab-ci/stages/manual.yml
-  - /.gitlab-ci/stages/scan.yml
-  - /.gitlab-ci/stages/deploy.yml
+  #
+  - /packages/code-du-travail-api/.gitlab-ci.yml
+  - /packages/code-du-travail-frontend/.gitlab-ci.yml
+  # - /packages/code-du-travail-nlp/.gitlab-ci.yml
+  - /packages/react-fiche-service-public/.gitlab-ci.yml
+  - /packages/react-ui/.gitlab-ci.yml
+  - /packages/sources/.gitlab-ci.yml
+  # - /.gitlab-ci/stages/quality.yml
+  # - /.gitlab-ci/stages/register.yml
+  # - /.gitlab-ci/stages/manual.yml
+  # - /.gitlab-ci/stages/scan.yml
+  # - /.gitlab-ci/stages/deploy.yml
   - /.gitlab-ci/stages/notify.yml
 
   - /optional/e2e/.runners/puppetteer/.gitlab-ci.yml
 
 stages:
   - "Prepare"
   - "Code Quality"
+  - "Build"
   - "Manual Deploy"
   - "Registration"
   - "Post Registration"
   - "Deploy"
   - "Notify Finished Deployment"
   - E2E test
+
+.prepare_stage:
+  stage: "Prepare"
+  dependencies: []
+
+.quality_stage:
+  stage: "Code Quality"
+  dependencies: []
+
+.register_stage:
+  stage: "Registration"
+  dependencies: []
+
+#
+
+Create namespace:
+  extends: .base_create_namespace_stage
+  stage: "Registration"
+  except:
+    - /^renovate/*
```

Comment:
this exclude renovate branch / Pr to trigger namespace creation ?

Reply:
I will remove this